packages/cloud-cognitive/src/components/CreateModal/_storybook-styles.scss

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright IBM Corp. 2020, 2020
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@import './index';
+@import 'carbon-components/scss/components/number-input/number-input';
+@import 'carbon-components/scss/components/text-area/text-area';
+@import 'carbon-components/scss/components/text-input/text-input';
+@import 'carbon-components/scss/components/dropdown/dropdown';
+@import 'carbon-components/scss/components/date-picker/date-picker';
+@import 'carbon-components/scss/components/radio-button/radio-button';
+@import 'carbon-components/scss/components/progress-indicator/progress-indicator';
+@import 'carbon-components/scss/components/tooltip/tooltip';
+@import 'carbon-components/scss/components/modal/modal';
+
+.bx--number {
+  min-width: 100%;
+}
+
+.bx--number__control-btn::before {
+  background-color: transparent;
+}
+
+.bx--number__control-btn::after {
+  background-color: transparent;
+}
+
+.bx--date-picker {
+  min-width: 100%;
+}
+
+.bx--date-picker-container {
+  min-width: 100%;
+}
+
+.bx--date-picker.bx--date-picker--single .bx--date-picker__input {
+  min-width: 100%;
+}
+
+.bx--list-box__menu-icon {
+  top: $spacing-03;
+}
+
+.example-wrapper {
+  display: grid;
+  grid-gap: $spacing-05;
+  grid-template-columns: 1fr 1fr;
+}
```

Comment:
I noticed these styles are affecting some elements inside the modal. This file will only showup on storybook and not the library code. Should these be permanent styles to those components being used inside the modal?

Reply:
Good catch! Technically, the C&CS guidelines for forms inside of a modal are that they take up 100% width of the component. However, `DatePicker` is the one component that is _not_ 100% width, and breaks this rule. Since it is a C&CS guideline, but not a Carbon one, I had to add this additional code. Because this component is a "dumb" component, and I did not want to explicitly add a width to any date pickers in it, as this could be completely up to design.

Reply:
Gotcha 👍🏻